### DISCUSSION TITLE
Why maxlenght arent working in my input?

### DISCUSSION THREAD

**Discussion by DennisWarchocki** [neuefische/questions#194] [Web] [answered]
Hi guys,

 I would like to limit my input field to a maximum of 40 characters and have used maxlenght for this. However, I still seem to be able to set an unlimited number of characters in my input field, and I don't understand why. Did I do something wrong or forgot? I would be happy if you could take a look at it. Thank you very much!

<img width="992" alt="Bildschirmfoto 2022-09-30 um 11 14 24" src="https://user-images.githubusercontent.com/109968361/193238232-533ced34-00f4-4756-9dd5-cf8bab0b8d79.png">
<img width="719" alt="Bildschirmfoto 2022-09-30 um 11 21 03" src="https://user-images.githubusercontent.com/109968361/193238444-0b3beadf-7908-475e-ab64-de839afe5f42.png">

Link to my Repo:
https://github.com/DennisWarchocki/capstone-project

**Comment by MerleMak** [comment, marked as answer]
Hi Dennis, 
Try using maxLength in camelCase, and see if that works :)
Let me know if it did :)

**Reply by DennisWarchocki** [reply]
It did! Thanks alot!!! :)

**Reply by MerleMak** [reply]
Perfect!
